v2/go.mod

```diff
@@ -7,6 +7,7 @@ replace github.com/brigadecore/brigade/sdk/v2 => ../sdk/v2
 require (
 	github.com/AlecAivazis/survey/v2 v2.0.7
 	github.com/Azure/go-amqp v0.13.1
+	github.com/Netflix/go-expect v0.0.0-20180615182759-c93bf25de8e8 // indirect
```

Comment:
Any idea how this new dependency got introduced?

Reply:
Oops yeah I remember. I was attempting to have a test confirm the prompt automatically but couldn't figure it out. I will delete this import now